incubator/tensorflow-inception/templates/inception-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
It seems like this job should be a pre/post-install hook. Actually, it might even be better as an init-container, but if we intend to scale the server out that would mean we would have multiple seed volumes.